ACPs/99-validatorsetmanager-contract/README.md

```diff
@@ -226,150 +191,82 @@ interface IACP99Manager {
 
-### IACP99SecurityModule
-
-Here is the proposed interface for the `IACP99SecurityModule` contract:
-
-```solidity
-/**
- * @notice Information about a validator registration
- * @param nodeID The NodeID of the validator node
- * @param validationID The ValidationID of the validation
- * @param weight The initial weight assigned to the validator
- * @param startTime The timestamp when the validation started
- */
-struct ValidatorRegistrationInfo {
-    bytes32 nodeID;
-    bytes32 validationID;
-    uint64 weight;
-    uint64 startTime;
-}
+## Backwards Compatibility
 
-/**
- * @notice Information about a change in a validator's weight
- * @param nodeID The NodeID of the validator node
- * @param validationID The ValidationID of the validation
- * @param nonce A sequential number to order weight changes
- * @param newWeight The new weight assigned to the validator
- * @param uptime The uptime information for the validator
- */
-struct ValidatorWeightChangeInfo {
-    bytes32 nodeID;
-    bytes32 validationID;
-    uint64 nonce;
-    uint64 newWeight;
-    ValidatorUptimeInfo uptimeInfo;
-}
+`ACP99Manager` is a reference specification. As such, it doesn't have any impact on the current behavior of the Avalanche protocol.
 
-/*
- * @title IACP99SecurityModule
- * @author ADDPHO
- * @notice The IACP99SecurityModule interface is the interface for the ACP99 security modules.
- * @custom:security-contact security@suzaku.network
- */
-interface IACP99SecurityModule {
-    error ACP99SecurityModule__ZeroAddressManager();
-    error ACP99SecurityModule__OnlyManager(address sender, address manager);
+## Reference Implementation
 
-    /// @notice Get the address of the ACP99Manager contract secured by this module
-    function getManagerAddress() external view returns (address);
+A reference implementation will be provided in Ava Labs' [ICM Contracts](https://github.com/ava-labs/icm-contracts/tree/main/contracts/validator-manager) repository. This reference implementation will need to be updated to conform to `ACP99Manager` before this ACP may be marked `Implementable`.
 
-    /**
-     * @notice Handle a validator registration
-     * @param validatorInfo The information about the validator
-     */
-    function handleValidatorRegistration(
-        ValidatorRegistrationInfo memory validatorInfo
-    ) external;
+### Example Integrations
 
-    /**
-     * @notice Handle a validator weight change
-     * @param weightChangeInfo The information about the validator weight change
-     */
-    function handleValidatorWeightChange(
-        ValidatorWeightChangeInfo memory weightChangeInfo
-    ) external;
-}
-```
+`ACP99Manager` is designed to be easily incorporated into any architecture. Two example integrations are included in this ACP, each of which uses a different architecture.
 
-### Reference Architecture
+#### Two-contract Design
```

Comment:
"Multi-contract Design" would be more generic (see the [BalancerValidatorManager](https://github.com/suzaku-network/suzaku-contracts-library/blob/main/src/contracts/ValidatorManager/BalancerValidatorManager.sol) that can have multiple security modules managing registrations/weight changes)
```suggestion
#### Multi-contract Design
```